frontend/src/lib/bare-metal-assets.ts

```diff
@@ -112,7 +85,7 @@ export async function syncBMAs(hosts: JsonArray, resources: JsonArray) {
     return { assets, errors }
 }
 
-export async function attachBMAs(assets: JsonArray, hosts: JsonArray, clusterName: string, errors: JsonArray) {
+export async function attachBMAs(assets: any[], hosts: any[], clusterName: string, errors: any[]) {
```

Comment:
Same comment here; we shouldn't be using `any`, you should type the structure of `assets` and `hosts`, and `errors`.  There's probably already types for these if you didn't transform the data into something else.

Reply:
fixed